### PR TITLE
create release branch later in cycle

### PR DESCRIPTION
**Pull request by jimangel**
Documentation necessary for the docs release lead to create the `release-x.xx` branch before the actual release instead of at the start of the release cycle.

This should reduce the amount of effort needed to sync `master` as well as reduce the probability of mistakes merging into release branches prematurely.

After this PR merges we will delete the `release-1.19` branch to be created closer to the release date.

/cc @savitharaghunathan @zacharysarah @kbarnard10

### CODE REVIEW

File: release-team/role-handbooks/docs/README.md [hunk 1]
```diff
@@ -457,12 +441,50 @@ $ git push origin config-toml-1.14
 
 ---
 
+#### Create the release branch
+
+Creating the release brach allows you to easily sunset the current docs under it's new branch `release-[current release]` after we merge `dev-[future release]`. For example if `master` represents `v1.20` you would create `release-1.20`.
```
Comment (zacharysarah) [inline release-team/role-handbooks/docs/README.md:446]:
```suggestion
Creating the release branch lets you snapshot the current docs in a new branch, `release-[current release]`, after merging `dev-[future release]`. For example: if `master` represents `v1.20`, you would create `release-1.20`.
```

File: release-team/role-handbooks/docs/README.md [hunk 2]
```diff
@@ -457,12 +441,50 @@ $ git push origin config-toml-1.14
...
+
+> Note: this requires someone with write access to `k/website` such as a Co-Chair.
```
Comment (zacharysarah) [inline release-team/role-handbooks/docs/README.md:448]:
```suggestion
> **Note:** Creating a branch requires someone with write access to `k/website`, such as a [SIG Docs co-chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership).
```

File: release-team/role-handbooks/docs/README.md [hunk 3]
```diff
@@ -457,12 +441,50 @@ $ git push origin config-toml-1.14
...
+$ git checkout --track upstream/release-[current release]
+$ git merge upstream/master
+$ git checkout -b merged-master-release-[current release]
+$ git commit -m "merged master into release-[current release] to keep in sync"
+$ git push origin merged-master-release-[current release]
+```
+
+#### Update Netlify
+
+Update the Netlify (contact a [SIG Docs chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) if you do not have access and they can assist with this):
```
Comment (zacharysarah) [inline release-team/role-handbooks/docs/README.md:472]:
```suggestion
Update the Netlify configuration. (A [SIG Docs chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) can assist you with access):
```

**Comment by onlydole** [comment]
+1 to @zacharysarah’s comments and otherwise looks good to me!

**Comment by jimangel** [comment]
Added nits and squashed, thanks @zacharysarah!

**Pull request merged**